packages/amplify-graphql-model-transformer/src/__tests__/model-transformer1.test.ts

```diff
@@ -0,0 +1,41 @@
+import { ModelTransformer } from '@aws-amplify/graphql-model-transformer';
```

Comment:
Same comment here about the test and snapshot filenames. Can we use something more descriptive than `model-transformer1`.